app/controllers/dishes_controller.rb

```diff
@@ -0,0 +1,12 @@
+class DishesController < ApplicationController
```

Comment:
Actually - don't think we will use and therefore need the "index" and "show" method for Dishes - since we will not have a standalone page in our app where all the dishes are listed, or where the details of 1 dish will be provided. Those elements will be embedded in the Show page of the Restaurant.